src/segments/umbraco.go

```diff
@@ -0,0 +1,231 @@
+package segments
+
+import (
+	"encoding/xml"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Umbraco struct {
+	props properties.Properties
+	env   platform.Environment
+
+	FoundUmbraco    bool
```

Comment:
As the context is already Umbraco, I would rename the following to:

```go
Found bool
Modern bool
```

And isn't Legacy == !Modern? If so we can use a single boolean.